config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,7 @@
+Sidekiq.configure_server do |config|
+  config.redis = {url: ENV.fetch("REDIS_URL", "redis:/localhost:6379")}
```

Comment:
Tại sao dùng dòng này

Reply:
Toàn bộ files này để xác định kết nối Sidekiq tới Redis ạ, đoạn này là config cho phía server